### PR TITLE
Sif tomb level

### PR DESCRIPTION
**Pull request by 1393F**
## About The Pull Request
adds Sif(from the citadel-based skyrat codebase)

adds a new tomb level with Sif(filled with other megafauna and necropolis loot protected by ice demonic miner)

adds the zweihander, a single-handed crusher thats ALSO in this level

## Why It's Good For The Game
probably isn't

## Changelog
:cl:
add: Adds a new tomb level
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Can't TM due to conflicts, when our current mining PRs are merged I'll tm this one

File: code/modules/tomb_worlds/loot.dm [hunk 1]
```diff
@@ -162,3 +162,282 @@
...
+	desc = "A glowing giant heavy blade that grows and slightly shrinks in size depending on the wielder's strength."
+	icon = 'ModularBungalow/mobs/megafauna/tomb/sif/icons/sif.dmi'
+	icon_state = "sword_of_the_forsaken"
+	lefthand_file = 'ModularBungalow/mobs/megafauna/tomb/sif/icons/item_lefthand.dmi'
+	righthand_file = 'ModularBungalow/mobs/megafauna/tomb/sif/icons/item_righthand.dmi'
+	w_class = WEIGHT_CLASS_HUGE
+	force = 25
+	throwforce = 10
+	block_chance = 10
+	armour_penetration = 80
```
Comment (EgorDinamit) [inline code/modules/tomb_worlds/loot.dm:216]: :raised_eyebrow:

**Comment by EgorDinamit** [comment]
Are you planning to make anything properly, or is it supposed to be 1:1 copy of necromancer's level?
image

Also, I'd like it if you didn't put chaos marine anywhere, but in space. There must be only one.

**Comment by 1393F** [comment]
> Are you planning to make anything properly, or is it supposed to be 1:1 copy of necromancer's level?

are you planning on using your words for actual changes to be made? Or are you just gonna continue saying shit that doesn't really tell me shit?

**Comment by EgorDinamit** [comment]
> > Are you planning to make anything properly, or is it supposed to be 1:1 copy of necromancer's level?
> are you planning on using your words for actual changes to be made? Or are you just gonna continue saying shit that doesn't really tell me shit?

Woah there, no need to be rude. I am just asking if you are planning to make a map for that tomb level.
Right now it's 1:1 copy of necromancer's one and using chaos marine anywhere outside of dedicated space ruin is 'no bueno'.

**Comment by 1393F** [comment]
> Woah there, no need to be rude. I am just asking if you are planning to make a map for that tomb level. Right now it's 1:1 copy of necromancer's one.

not good at mapping so no

**Comment by Kitsunemitsu** [comment]
As much as I like the chaos marine in space only, This has caused many issues in the past with administration and balance.
Mostly by miners rushing it and cheesing it roundstart. 

I would still like it removed from here, if you don't want to, I can do it for you.

**Comment by EgorDinamit** [comment]
> As much as I like the chaos marine in space only, This has caused many issues in the past with administration and balance. Mostly by miners rushing it and cheesing it roundstart.

Sounds like something we should be fixing. The only cheese I see miners doing is z-level abuse.
To solve it we could add some check for the z-level switch during combat, so miners wouldn't feel safe doing it.

File: code/modules/tomb_worlds/loot.dm (hunk 1 above)
Comment (EgorDinamit) [reply code/modules/tomb_worlds/loot.dm:216]: Yeah, that's not good, too much AP imo.

**Comment by Kitsunemitsu** [comment]
Alright, I want this.

**Comment by Kitsunemitsu** [comment]
Adding into a future PR.

**Pull request closed without merging**